app/assets/javascripts/views/searches/preview.js

```diff
@@ -245,14 +246,47 @@
       templateCompiled.pagination = htmlPagination;
 
       tableBuilder.render(templateCompiled, '#paginated-results', ".js-pagination");
-      
+
       if(navbarOn){
         enableNavbar(json.type);
       }
-      
+
     }, 500);
   }
 
+  function setDataResultGroups(json){
```

Comment:
This and the next function are not used anywhere in the code. It is not clear to me why they are here.